pkg/cloudprovider/metrics/cloudprovider.go

```diff
@@ -47,8 +52,22 @@ var methodDurationHistogramVec = prometheus.NewHistogramVec(
 	},
 )
 
+var (
+	errorsTotalCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: "cloudprovider",
+			Name:      "errors_total",
+			Help:      "Total number of errors returned from CloudProvider calls.",
+		},
+		[]string{
+			metricLabelController,
```

Comment:
I think we need to define both the `controller` label here as well as the `error_type` label

Reply:
indeed